docs/dev/building-on-zksync/hello-world.md

```diff
@@ -137,16 +137,6 @@ export default async function (hre: HardhatRuntimeEnvironment) {
   const greeting = "Hi there!";
   const deploymentFee = await deployer.estimateDeployFee(artifact, [greeting]);
 
-  // ⚠️ OPTIONAL: You can skip this block if your account already has funds in L2
```

Comment:
Do we need to also remove from the hardhat template? https://github.com/matter-labs/zksync-hardhat-template/blob/main/deploy/deploy-greeter.ts#L31-L39

Reply:
Resolving conversation, this shouldn't block the PR